### PR TITLE
[ml] Add a starter page for docstrings

### LINKED ISSUE #23296: [ml] Get the docstrings onto somewhere accessible by users
Right now, there's no way to access the docstrings. We need a way to access from the top-level documentation (without doing a large refactor, etc)

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?

So two things don't work yet:

1. ~Preprocessor doesn't build properly~
2. ~Tuner doesn't import properly.~

This page is ONLY for accessing docstrings and for iterating on the messaging. 

The intent is to express that this is ALPHA. We can orphan the page if necessary. 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/_toc.yml [hunk 1]
```diff
@@ -106,6 +106,8 @@ parts:
           - file: workflows/package-ref
       - file: ray-more-libs/index
         title: More Ray ML Libraries
+      - file: ray-air/getting-started
```
Comment (ericl) [inline doc/source/_toc.yml:109]: How about we nest it under "More Ray ML Libraries" for now?

File: doc/source/ray-air/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+
+Serving
+~~~~~~~
+
+.. automodule:: ray.serve.model_wrappers
+    :members:
+
+
+General utilities
```
Comment (ericl) [inline doc/source/ray-air/getting-started.rst:79]: Should split this up into "Results" and "Configs"

**Review by ericl — APPROVED** [review]

File: doc/source/_toc.yml (hunk 1 above)
Comment (maxpumperla) [reply doc/source/_toc.yml:109]: that's actually a good idea. makes it easier to surface stuff later

Comment (richardliaw) [reply doc/source/_toc.yml]: done

**Comment by richardliaw** [comment]
cc @amogkam , some trainer docstring changes for formatting.

**Comment by jianoaix** [comment]
Cherrypicked to 1.12.0rc1.

**Pull request merged**